README.md

```diff
@@ -89,17 +89,22 @@ If you would like to tail the logs in the console then you remove the detach fla
 
 The following are examples of some common Django management commands that you may need to run.
 
-* Make Migrations: `docker-compose run --rm manage makemigrations`
-* Merge Migrations: `docker-compose run --rm manage makemigrations --merge`
-* Run Migrations: `docker-compose run --rm manage`
-* Test: `docker-compose run --rm manage test`
+* Make Migrations: `docker-compose run --rm app ./manage.py makemigrations`
+* Merge Migrations: `docker-compose run --rm app ./manage.py makemigrations --merge`
```

Comment:
What does `--merge` do? Is this the right documentation? https://docs.docker.com/engine/reference/commandline/app_merge/

Reply:
I just cargo culted it from the existing docs which were added in https://github.com/codebuddies/backend/commit/f85bc3b81960542c32152a0124daf6b18cf8281b
The right docs are for that flag are: https://docs.djangoproject.com/en/3.0/ref/django-admin/#cmdoption-makemigrations-merge
Its a flag to do with squashing and merging migrations, but I'm not 100% sure why that specific flag is documented in this README.

Reply:
Ah, thank you for finding the docs for that `--merge` command! 🤔